metricbeat/module/elasticsearch/testing.go

```diff
@@ -0,0 +1,70 @@
+// +build !integration
```

Comment:
We could also move this common testing code to its own package (under `metricbeat/module/elasticsearch/testing/`) so it is only imported in tests and we don't need to add build tags. I was thinking about doing something like this for the RabbitMQ test server.

Reply:
+1 on moving it in it's own package in the future and also generalise it. There is already a `testing` package in Metricbeat. But before doing that i would check if the approach here fits other modules and have at least 2-3 more similar implementation to then take it out.